<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_re_check">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>640</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>320</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 255, 237);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="skip_btn2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>311</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>421</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>311</width>
     <height>421</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 20pt &quot;나눔스퀘어 ExtraBold&quot;;
background-color: rgb(176, 181, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="re_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>311</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>421</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>311</width>
     <height>421</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 20pt &quot;나눔스퀘어 ExtraBold&quot;;
background-color: rgb(213, 232, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="plz_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>1366</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>plz_bg</zorder>
  <zorder>label_re_check</zorder>
  <zorder>skip_btn2</zorder>
  <zorder>re_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
